Jenkinsfile

```diff
@@ -66,6 +70,96 @@ node {
           rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
         """
       }
+
+      // set akamai fast purge env
+      openShiftUtils.withJnlpNode(        
+        image: "quay.io/redhatqe/origin-jenkins-agent-akamai:4.9",
+        namespace: "insights-dev-jenkins"
+      ) {
+        sh "wget https://raw.githubusercontent.com/RedHatInsights/cloud-services-config/${ENVSTR}-${RELEASESTR}/akamai/cache_buster/bust_cache.py"
+        // get akamai fast purge credentials
+        withCredentials([file(credentialsId: "jenkins-eccu-cache-purge", variable: 'EDGERC')]) {
+          sh "python3 bust_cache.py $EDGERC ${BRANCH} ${NAVLIST}"
+        }
+        // we have to wait 5-10s until akamai fast purge takes effect
+        sleep(10)
+      }
+    }
+  }
+
+  stage ("Run Akamai Production Smoke Tests") {
+    try {
+      if (RUN_SMOKE_TESTS) {
+        openShiftUtils.withNode {
```

Comment:
```groovy
openShiftUtils.withNode(
    image: "quay.io/cloudservices/iqe-tests:latest""
) {
```

Reply:
@quarckster are the `akamai` and `3scale` plugin images already included in the `iqe-tests:latest` image? 

Right now, this is just using the iqe-core image.

Reply:
@john-dupuy yes, they are included https://url.corp.redhat.com/1d30796

Reply:
Yes, sound good to me.

Reply:
Works for me!